orderer/common/cluster/clusterservice_test.go

```diff
@@ -592,6 +587,31 @@ func TestConfigureNodeCerts(t *testing.T) {
 		require.NoError(t, err)
 		require.Equal(t, []byte(nil), svc.MembershipByChannel["mychannel"].MemberMapping[authRequest.FromId])
 	})
+
+	t.Run("Configure remove streams based on the new consenters set", func(t *testing.T) {
```

Comment:
This test is too reliant on implementation details, let's remove it

Reply:
removed the testcase